src/main/groovy/com/profitbricks/sdk/model/Volume.groovy

```diff
@@ -27,7 +27,13 @@ final class Volume extends ModelBase {
     int deviceNumber
 
     @Override
-    final Volume create() { (super.create() as Volume)?.with dataCenter }
+    final Volume create() {
+        if(image!=null)
```

Comment:
in the SDKExample.groovy i tried to create a volume and providing an image id, in that case because you are giving the `licenceType = 'UNKNOWN'` a default value the request fails.

The other solution was to remove the default value but my initial thought was it must be there for a reason.